docs/features/impact-radius.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Impact Radius
+icon: material/adjust
+---
+
+**Impact Radius** helps you analyze changes and identify downstream impacts at the column level.
+
+While dbt provides a similar capability using the [state selector](https://docs.getdbt.com/reference/node-selection/methods#state) with `state:modified+` to identify modified nodes and their downstream dependencies, Recce goes further. By analyzing SQL code directly, Recce enables **fine-grained impact radius analysis**. It reveals how changes to specific columns can ripple through your data pipeline, helping you prioritize which models—and even which columns—deserve closer attention.
+
+
+=== "state:modified+"
+    
+    ![Breaking Change Analysis (disabled)](../assets/images/features/impact-radius-legacy.png){: .shadow}
+
+=== "Impact Radius"
+    
+    ![Breaking Change Analysis](../assets/images/features/impact-radius.png){: .shadow}
+
+
+
+## Usage
+
+### Show impact radius
+
+1. Click the **Impact Radius** button in the upper-left corner.
+
+    ![Impact Radius button highlighted](../assets/images/features/impact-radius-1.png){: .shadow}
+
+1. The impact radius will be displayed.
+
+    ![Impact radius displayed on screen](../assets/images/features/impact-radius-2.png){: .shadow}
+
+1. To exit impact radius view, click the close button in the upper-left corner.
+    
+    ![Close button for exiting impact radius view](../assets/images/features/impact-radius-3.png){: .shadow}
+
+### Show impact radius for a single changed model
+
+1. Hover over a changed model, then click the **target icon** or right-click the model and click the **Show Impact Radius**
+
+    ![Target icon for showing impact radius of a single model](../assets/images/features/impact-radius-single-1.png){: .shadow}
+
+1. The impact radius for this model will be displayed.
+
+    ![Impact radius for a single model displayed on screen](../assets/images/features/impact-radius-single-2.png){: .shadow}
+
+1. To exit impact radius view, click the close button in the upper-left corner.
+    
+    ![Close button for exiting single model impact radius view](../assets/images/features/impact-radius-single-3.png){: .shadow}
+
+
+## How it works
+
+Two core features power the impact radius analysis:
+
+**[Breaking Change Analysis](./breaking-change-analysis.md)** classifies modified models into three categories:
```

Comment:
These description are mixing breaking change and impact radius. We just discussed that it's the modified models introduce breaking change, the scope is the modified model itself. See https://www.notion.so/infuseai/Building-Impact-Radius-2-The-Technical-Breakthrough-22479451d35780d89b76e1dfef876715?source=copy_link#22679451d35780eaac6ccf3a7159e30a

Reply:
Updated